src/int.js

```diff
@@ -121,18 +128,22 @@ Sk.builtin.int_ = function (x, base) {
     return this;
 };
 
+Sk.abstr.setUpInheritance("int", Sk.builtin.int_, Sk.builtin.numtype);
+
 Sk.builtin.int_.co_varnames = [ "base" ];
 Sk.builtin.int_.co_numargs = 2;
-Sk.builtin.int_.$defaults = [ new Sk.builtin.int_(10) ];
```

Comment:
Ah ok, you don't want to keep it close to the method? There are some more here: https://github.com/skulpt/skulpt/blob/master/src/sorted.js#L56